csharp-basics/exercises/ClassesAndObjects/Exercise8/SavingsAccount.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise8
+{
+    public class SavingsAccount
+    {
+        private double _annualInterestRate;
+
+        public SavingsAccount(double accountStartingBalance)
+        {
+            Balance = accountStartingBalance;
+        }
+
+        public int AccountTerm { get; private set; }
+
+        public double TotalDeposited { get; private set; }
+
+        public double TotalWithdrawn { get; private set; }
+
+        public double Balance { get; private set; }
```

Comment:
Tu viņu izmanto priekš aprēķiniem, kā tev viņš var būt public?

Reply:
Edgars atļava izmantot private set. Man slinkums to pārtaisīt, bet citus uzdevumus taisīšu kā tu teici